client/jetpack-cloud/sections/partner-portal/payment-methods-v2/index.tsx

```diff
@@ -18,9 +19,12 @@ export default function PaymentMethodListV2() {
 		<Layout className="payment-method-list" title={ title } wide>
 			<LayoutTop>
 				<LayoutHeader>
-					<Title>{ title } </Title>
-					<Subtitle>{ subtitle }</Subtitle>
+					<CardHeading size={ 36 }>{ title }</CardHeading>
+					<Button href="/partner-portal/payment-methods/add" primary>
+						{ translate( 'Add new card' ) }
+					</Button>
```

Comment:
We could use `LayoutHeaderActions` to handle this. You can use this [file](https://github.com/Automattic/wp-calypso/blob/db5e20798888f32a080aeaa292ae607b0199a1e5/client/jetpack-cloud/sections/partner-portal/issue-license-v2/index.tsx) as reference